perl-package/AI-MXNet/lib/AI/MXNet/NDArray/Sparse.pm

```diff
@@ -80,10 +80,100 @@ use overload '""' => sub {
                         my $shape_info = join('x', @{ $self->shape });
                         sprintf("\n<%s, %s @%s>", $self->_class_name, $shape_info, $self->context);
                      },
+             '+'  => \&add,
+             '-'  => \&subtract,
+             '*'  => \&multiply,
+             '/'  => \&divide,
              '+=' => \&not_implemented,
              '-=' => \&not_implemented,
              '*=' => \&not_implemented,
              '/=' => \&not_implemented;
+
+method add(AI::MXNet::NDArray|Num $other, $reverse=)
+{
```

Comment:
I really appreciate your fast response @sergeykolychev thanks for unblocking us. While I'm not a perl user but some documentations for the new function will be great. Maybe worth adding next time

Reply:
Yeah, I'll next time. Tried to unblock quickly. Though I don't see anybody using the functions directly and not via the overloaded operators.